tests/testthat/test-get_measurements.R

```diff
@@ -1,5 +1,8 @@
 context("get_measurements from data objects")
 
+source("helper-organismdb.R")
+source("helper-makeEset.R")
```

Comment:
@hcorrada I source the helper files. I did not find any documentation on this change.